bench/workgen/workgen.h

```diff
@@ -437,14 +448,18 @@ struct WorkloadOptions {
 struct Workload {
     WorkloadOptions options;
     Stats stats;
+    WT_CONNECTION *_connection;
```

Comment:
I don't think `WT_CONNECTION` is needed in the `Workload` class.  There are a couple reasons for this. One is there shouldn't be any use of the connection before the `run` statement.  Second is a design issue: a `Workload` is kind of like a "template" for the work to be performed.  (I'm not using "template" in the C++ meaning).  There's no reason why a Workload can't be created and run with one connection, and then run with a different one.  If you need access to the connection at runtime (in the timing thread?), I would suggest passing it as an argument to your thread when the thread is created.

Reply:
Removed `WT_CONNECTION` from `Workload` class and passing it as an argument to `WorkloadRunner::run_all` function.